app-catalog/src/helpers/catalog.ts

```diff
@@ -0,0 +1,89 @@
+import { fetchCatalogs } from '../api/catalogs';
+
+const ANNOTATION_URI = 'catalog.ocne.io/uri';
```

Comment:
I guess these are specific to OCNE. Is there some standard place this data could be gotten from that could be used instead? If not that's fine.

Reply:
@muraliinformal see this question^

Reply:
Yeah, I can’t think of anything. We have the headlamp.dev domain name so catalog.headlamp.dev/uri ?